### ISSUE TITLE
[Help] `--output-filename` shows incorrect default filename

### ISSUE THREAD

**Issue by bersbersbers** [Nuitka/Nuitka#2677]
Windows
```
    --output-filename=FILENAME
                        Specify how the executable should be named. For
                        extension modules there is no choice, also not for
                        standalone mode and using it will be an error. This
                        may include path information that needs to exist
                        though. Defaults to '<program_name>' on this platform.
                        .exe
```

Linux:
```
    --output-filename=FILENAME
                        Specify how the executable should be named. For
                        extension modules there is no choice, also not for
                        standalone mode and using it will be an error. This
                        may include path information that needs to exist
                        though. Defaults to '<program_name>' on this platform.
                        .bin
```

I think the last lines should be
> Defaults to '<program_name>.ext' on this platform.

(i.e., with the extension inline rather than appended)

This is with nuitka 2.0.0.

**Comment by kayhayen** [comment]
Thanks for your report, this should be old. A missing brace was causing formatting to go wrong. This will be fixed in 2.0.1, closing early for lack of general relevance.

**Comment by kayhayen** [comment]
This is part of the hotfix release 2.0.1 that I just made.
